config/scheduler-chromeos.yaml

```diff
@@ -128,6 +128,71 @@ scheduler:
   - job: tast-basic-x86-stoneyridge
     <<: *test-job-stoneyridge
 
+  - job: tast-decoder-v4l2-sl-av1-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms: &mediatek-av1-platforms
+      - mt8195-cherry-tomato-r2
+
+  - job: tast-decoder-v4l2-sl-h264-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms:
+      - mt8183-kukui-jacuzzi-juniper-sku16
+      - mt8192-asurada-spherion-r0
+      - mt8195-cherry-tomato-r2
+
+  - job: tast-decoder-v4l2-sl-hevc-arm64-mediatek
+    <<: *test-job-mediatek
+    platforms: *mediatek-av1-platforms
```

Comment:
Can we just list the platforms instead of using the labels? Afaik there's no guarantee that a platform supporting hevc also supports av1, so I'd rather list the platforms individually for each suite if possible.

Reply:
Done (mostly), I'm keeping anchors for VP9 tests as they all run on the same devices